### PR TITLE
Create a Guide for StackBlitz

### PR DESCRIPTION
**Pull request by adamjarling**
First draft pass at a Guide/Documentation for running StackBlitz.  Maybe there's a better spot to highlight this?  Also not really sure how in depth to go here...  

Curious on your two cents @markpbaggett (not necessarily today as it's getting late), since this StackBlitz environment may be fresh/new to your eyes.

### CODE REVIEW

File: pages/stackblitz.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+<Callout type="info">
+  When demoing in StackBlitz, your are in `dev` mode when configuring
+  `/config/canopy.json`. This means that you can use the `dev` settings to
+  target a IIIF Collection or Manifest.
+</Callout>
+
+For example, let's use the [Charlotte Mormon Collection](https://api.dc.library.northwestern.edu/api/v2/search?query=collection.id:%22b46c9cf7-0e49-4993-9e0e-3cda030c4439%22&as=iiif) from Northwestern University Libaries.
+
+```json file="/config/canopy.json" {19-20}
```
Comment (markpbaggett) [inline pages/stackblitz.mdx:50]:
```suggestion
```json filename="/config/canopy.json" {19-20}
```

**Review by markpbaggett — CHANGES_REQUESTED** [review]
I stand by what I said previously, but just noticed this small error.

**Comment by adamjarling** [comment]
@markpbaggett Committed your suggestion in the PR.  Thanks!

File: pages/get-started.mdx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import { Box, Text, Section } from "@radix-ui/themes";
   </Text>
 </Section>
 
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz_small.svg)](https://stackblitz.com/fork/github/canopy-iiif/canopy-iiif?startScript=dev)
```
Comment (mathewjordan) [inline pages/get-started.mdx:14]: 👍

**Comment by mathewjordan** [comment]
Gonna merge this im so we can have the docs at a checkpoint.

**Pull request merged**